Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN if [ "$CANTALOUPE_VERSION" = 'dev' ] ; then \
     fi
 
 # The second stage of our multi-stage build builds the kakadu libraries and binaries
-FROM ubuntu:18.10 AS KAKADU_TOOL_CHAIN
+FROM ubuntu:19.10 AS KAKADU_TOOL_CHAIN
```

Comment:
Not necessarily an issue, but is there a reason we're going with the *.10 variants instead of the *.04?

Reply:
We had made that decision at a point where it looked like the LTS' packages weren't maintained past a certain point (except for paying customers), way before their expiration perio. I'm not sure that conclusion is actually true now though.

Reply:
It may be the distinction between main and universe packages though (and we were using universe packages).